components/root/root.jsx

```diff
@@ -142,6 +142,17 @@ export default class Root extends React.Component {
         };
     }
 
+    handleTabKey = (e) => {
+        if (isKeyPressed(e, Constants.KeyCodes.TAB)) {
+            const activeElement = e.target;
+            activeElement.classList.add('keyboard-focus');
+
+            activeElement.addEventListener('blur', () => {
+                activeElement.classList.remove('keyboard-focus');
```

Comment:
Can you also remove the event listener here? Otherwise we're going to stack up a ton of event listeners. See https://stackoverflow.com/a/4402359 for an example on how to do that

Also if you could remove the event listener before adding it every time that would ensure you're adding multiple event listeners

Reply:
@jwilander We don't need to remove the event listener here because we're passing `{once: true}` into `addEventListener` which causes it to automatically remove itself